### PR TITLE
Handle Android Feature module

### PR DESCRIPTION
**Pull request by CedrickFlocon**
Hi,

With the Android Plugin 3.0 + android added an other plugin type [`com.android.feature`](https://developer.android.com/topic/instant-apps/getting-started/structure.html)

I am not an expert of gradle plugin but this should do the job.
Waiting for your feedback

Best

### CODE REVIEW

**Comment by CedrickFlocon** [comment]
@henryju I just update `Add test for ...` commit with the properties assertions

File: integrationTests/src/test/java/org/sonarqube/gradle/AndroidTest.java [hunk 1]
```diff
@@ -204,6 +204,22 @@ public void testingBlueprint_default_flavor() throws Exception {
     assertThat(props.getProperty(":module-android-library.sonar.java.source")).isEqualTo("1.7");
     assertThat(props.getProperty(":module-android-library.sonar.java.target")).isEqualTo("1.7");
 
+    // Feature module has no local tests
+    assertThat(stream(props.getProperty(":module-android-library.sonar.sources").split(",")).map(Paths::get))
```
Comment (henryju) [inline integrationTests/src/test/java/org/sonarqube/gradle/AndroidTest.java]: Module key is `:module-android-feature`

File: integrationTests/src/test/java/org/sonarqube/gradle/AndroidTest.java [hunk 2]
```diff
@@ -204,6 +204,22 @@ public void testingBlueprint_default_flavor() throws Exception {
...
+            .containsOnly(
+                    baseDir.resolve("module-android-library/src/main/java"),
```
Comment (henryju) [inline integrationTests/src/test/java/org/sonarqube/gradle/AndroidTest.java]: Module baseDir starts with `module-android-feature`

**Comment by CedrickFlocon** [comment]
My bad I thought I had updated it

**Comment by CedrickFlocon** [comment]
Any update @henryju ?

**Comment by henryju** [comment]
Changes applied, but I don't know when we'll have time to release. Anyway thanks for the contribution (and the ping :) )

**Comment by henryju** [comment]
I reverted the merge, since IT is failing with:
```
22:48:10 A problem occurred configuring project ':module-android-feature'.
22:48:10 > Feature modules require compileSdkVersion set to 26 or higher. compileSdkVersion is set to 25
```
Please fix and reopen the PR (if possible) or submit a new one.

**Pull request merged**